packages/preact/src/index.ts

```diff
@@ -206,8 +206,15 @@ hook(OptionsTypes.RENDER, (old, vnode) => {
 		typeof window !== "undefined" &&
 		window.__PREACT_SIGNALS_DEVTOOLS__
 	) {
+		let parentComponent: VNode | undefined = vnode.__;
+		while (parentComponent && typeof parentComponent.type === "string") {
+			parentComponent = parentComponent.__;
+		}
 		window.__PREACT_SIGNALS_DEVTOOLS__.enterComponent(
-			vnode.type.displayName || vnode.type.name || "Unknown"
+			(parentComponent && typeof parentComponent.type !== "string"
```

Comment:
this code is a bit messy right now. if we actually want this feature ill try tidy it up